init/cmdline.go

```diff
@@ -192,7 +192,14 @@ func parseParams(params string) error {
 			if err != nil {
 				return fmt.Errorf("root=%s: %v", value, err)
 			}
+		case "modules":
+			if (value == "") {
+				break
+			}
 
+			for _, o := range strings.Split(value, ",") {
+				config.ModulesForceLoad = append(config.ModulesForceLoad, o)
```

Comment:
I think shorter version should work here as well

```go
modules := range strings.Split(value, ",")
config.ModulesForceLoad = append(config.ModulesForceLoad, modules...)
```

Reply:
Thanks, good idea. Adjusted accordingly. Haven't tested it with this change but I assume it should work. On a side note: the `if value == ""` is necessary as otherwise Booster would attempt to load `/usr/lib/modules/.ko` with a `modules=` kernel parameter.